### DISCUSSION TITLE
Cannot access website - HTTPS security

### DISCUSSION THREAD

**Discussion by crcardenas** [iqtree/iqtree2#363] [General]
Hello,

I know this is slightly off topic, but relevant given the great documentation on the website. But basicallyhe IQTree2 website is inaccessible using firefox, opera, or midori browsers. There seems to be an issue with your website security token:

```
Error code: SSL_ERROR_UNSUPPORTED_VERSION

    The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
    Please contact the website owners to inform them of this problem.

This website might not support the TLS 1.2 protocol, which is the minimum version supported by Firefox.
```

They all indicate an error similar too `ERR_SSL_VERSION_OR_CIPHER_MISMATCH` 

This seems to be a reoccurring issue (https://github.com/iqtree/iqtree2/discussions/237); and I've noticed for over a year that the website has always been flagged as insecure (but accessible) by my browser..

If anyone else lands here, thankfully most of the documentation can be found on github (https://github.com/iqtree/iqtree2/wiki).

**Comment by crcardenas** [comment]
I've found a temporary work around with Firefox that I really don't like.But because I'm running some analyses I would like to refer to some docs on the website that aren't on the GitHub wiki (e.g., the concordance factor references).

1) Open a new tab in Firefox enter `about:config` as an address, click "accept the risk and proceed with caution"
2) search `security.tls.version.min`in the config preferences
3) click edit and change the value of `security.tls.version.min` from 3 to 1
4) You should still get a "HTTPS-Only mode" alert, informing you that the IQTree2 website is not secure but you should now have the option to click "continue to the HTTP site"
5) once you are done browsing iqtree.org you need to repeat **steps 1** and **2**
6) click edit and change the value of `security.tls.version.min` from 1 to 3

Hopefully the website maintainers can solve this issue. But if not, hopefully this work around suffices for now; and into the future.

**Comment by roblanf** [comment]
@crcardenas We'll prioritise looking at the website issue ASAP, but likely it will be next year, since we're all up against end of year deadlines here for now. 

However, the website just mirrors the wiki, so everything on the website is on the wiki somewhere. 

E.g. all the mentions of 'concordance' on the wiki are here:

https://github.com/search?q=repo%3Aiqtree%2Fiqtree2+concordance&type=wikis

Hope that helps a little

**Comment by roblanf** [comment]
Also, I just checked, and I can load the iqtree website no problem on the latest version of firefox.

Make sure you are using http, not https, and it should be fine.

i.e. use: 

http://www.iqtree.org/

not

https://www.iqtree.org/

**Reply by crcardenas** [reply]
Rob,

Thanks for getting back so quickly! Okay, so I didn't think to check the http vs https. That is super helpful, I do not get any kind of error navigating to the http wepsite!

**Reply by roblanf** [reply]
no problem! Despite the simple solution, I'll open an issue to highlight that we should do something to make sure the website also works with https, or at least redirects to http or something (I don't know how any of that works, but presumably it's not beyond us).

**Reply by roblanf** [reply]
Issue created (https://github.com/iqtree/iqtree2/issues/368), so I'll close this discussion for now. We'll fix the https thing when we can! 

And thanks for highlighting it @crcardenas
